src/BlazorWebView/src/Maui/Standard/BlazorWebViewHandler.cs

```diff
@@ -14,5 +15,13 @@ public partial class BlazorWebViewHandler : ViewHandler<IBlazorWebView, object>
 
 		/// <inheritdoc cref="IBlazorWebView.CreateFileProvider" />
 		public virtual IFileProvider CreateFileProvider(string contentRootDir) => throw new NotSupportedException();
+
+		/// <summary>
+		/// Calls the specified <paramref name="workItem"/> asynchronously and passes in the scoped services available to Razor components.
+		/// </summary>
+		/// <param name="workItem">The action to call.</param>
+		/// <returns>Returns a <see cref="Task"/> representing <c>true</c> if the <paramref name="workItem"/> was called, or <c>false</c> if it was not called because Blazor is not currently running.</returns>
+		/// <exception cref="ArgumentNullException">Thrown if <paramref name="workItem"/> is <c>null</c>.</exception>
+		public virtual Task<bool> TryDispatchAsync(Action<IServiceProvider> workItem) => throw new NotSupportedException();
```

Comment:
Are we going to have cases where you need to do async work here? We might as well take a `Func<IServiceProvider, Task>` to begin with if we think we might end up there at some point?

Reply:
We discussed numerous options in this comment thread (hidden by default): https://github.com/dotnet/aspnetcore/pull/46644#pullrequestreview-1299343855

Though I didn't see any mention of the work item itself being async...

So I guess we'll have to see if anyone wants that version as well.